lib/testers/starter-code-tester.ts

```diff
@@ -67,6 +67,10 @@ export default class StarterCodeTester extends BaseTester {
 
     Logger.logSuccess("Script output verified");
 
+    // Commmit newly copied ./you_program.sh file from previous step
+    await exec(`git -C ${this.copiedStarterDir} add .`);
+    await exec(`git -C ${this.copiedStarterDir} commit -m "Copy .codecrafters/run.sh -> your_program.sh"`);
```

Comment:
This will ignore ALL changes that we're supposed to be checking for. Instead only commit your_program.sh if it's changed